### PR TITLE
New Zh(Chinese) Source

### PR DESCRIPTION
**Pull request by Nabeelshar**

### CODE REVIEW

File: app/src/main/java/my/noveldokusha/scraper/sources/Aixdzs.kt [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        }
+    }
+
+    override suspend fun getCatalogList(index: Int): Response<PagedList<BookMetadata>> {
+        val page = index + 1
+//        val url = baseUrl.toUrlBuilderSafe().apply {
+//            if (page == 1) addPath("all.html")
+//            else addPath("all-$page.html")
+//        }
+
```
Comment (nanihadesuka) [inline app/src/main/java/my/noveldokusha/scraper/sources/Aixdzs.kt:50]: Remove commented code.

Comment (Nabeelshar) [reply app/src/main/java/my/noveldokusha/scraper/sources/Aixdzs.kt:50]: that code is reminder to add next page with that setting.

**Comment by nanihadesuka** [comment]
Please next time do pull request with custom branch name, not generic "master"

**Comment by nanihadesuka** [comment]
Both `getCatalogSearch`  and `getCatalogList` don't use  `index`. Why? If the webpage has more results these functions must returns them also.

**Review by nanihadesuka — CHANGES_REQUESTED** [review]
Need to use index or explicitly tell only the first page can be retrieved (which I doubt).

**Pull request closed without merging**